code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -206,6 +206,8 @@
 		bust_open()
 
 /obj/structure/closet/attackby(obj/item/W, mob/user, params)
+	if (!istype(src, /obj/structure/closet/bsdroppod))
+		return ..()//dont wanna do anything if you're a drop pod

```

Comment:
this seems like the opposite of what you want

Reply:
i have brain cells i swear